Neos.Flow/Classes/Mvc/ActionResponse.php

```diff
@@ -191,12 +191,17 @@ public function getStatusCode(): int
         return $this->statusCode ?? 200;
     }
 
+    public function hasContentType(): bool
```

Comment:
We should start removing useless docblocks that only repeat the existing PHP typehints